numpy/lib/function_base.py

```diff
@@ -4054,6 +4054,10 @@ def trapz(y, x=None, dx=1.0, axis=-1):
     4.0
     >>> np.trapz([1,2,3], x=[4,6,8])
     8.0
+    >>> np.trapz([0,1,0], x=[0,1,2])
+    1.0
+    >>> np.trapz([0,0,1], x=[0,2,1])
+    -0.5
```

Comment:
Perhaps more insightful would be to repeat the example above with the x coordinates in reverse order, and to deliberately integrate a closed curve:

```suggestion
    # integrate in reverse
    >>> np.trapz([1,2,3], x=[8,6,4])  
    -8.0

    # Find the area of a circle
    >>> theta = np.linspace(0, 2 * np.pi, num=1000, include_endpoint=True)
    >>> np.trapz(np.cos(theta), x=np.sin(theta))
    3.141571941375841
```

Reply:
Nice intuitive examples!